packages/lockfile-file/src/read.ts

```diff
@@ -49,10 +49,10 @@ export async function readWantedLockfile (
   }
 ): Promise<Lockfile | null> {
   const lockfilePath = path.join(pkgPath, WANTED_LOCKFILE)
-  return (await _read(lockfilePath, pkgPath, opts)).lockfile
+  return (await read(lockfilePath, pkgPath, opts)).lockfile
 }
 
-async function _read (
+export async function read (
   lockfilePath: string,
   prefix: string,
```

Comment:
rename `prefix` to `pkgPath`?

Reply:
ok

and rename the function to `readLockfile`